lib/prorate/rate_limit.lua

```diff
@@ -49,7 +50,7 @@ if (new_bucket_level + n_tokens) <= max_bucket_capacity then
   retval = {0, math.ceil(new_bucket_level)}
 else
   redis.call("SETEX", block_key, block_duration, now + block_duration)
-  retval = {block_duration, 0}
+  retval = {tostring(block_duration), -1}
```

Comment:
Nice change, can you please check or add a test to ensure that the [limit](https://github.com/WeTransfer/prorate/blob/master/lib/prorate/throttle.rb#L105) behavior was not affecter?

Reply:
Thanks :) 
I'm quite confident that the limit behavior is unaffected (except for when it needs to be affected to solve the bug) : bucket_level is set to -1 if and only if the throttler needs to block, in which case an error is thrown an no limit is returned. I don't think there's anything to add to the existing tests, as there's nothing new to test?

Reply:
The line I linked will return `@limit - (-1)` now which is even higher than the bucket capacity. Use cases in our apps don't rely on the `throttle!`, I am not sure that this is not going to change other users' expectations. If you're confident that it's wont, let's at least bump the minor version to draw attention to the change when people will update their dependencies?

Reply:
It won't actually ever return @limit - (-1), since if bucket_level == -1 an error will be raised. So I don't think we really need to bump the minor version, but I don't mind.

Reply:
That's fair, thanks for pointing out!